lab1.js

```diff
@@ -133,7 +158,17 @@ var tooHungryDay;
  meals)
 */
 
-// your code goes here
+var sum = 0;
+
+for (var i = 0; i < mealsPerDay.length; i++) {
+  sum += mealsPerDay[i];
+}
+
+var average = sum / mealsPerDay.length;
+var x = 0;
+if (average < 4) {
```

Comment:
this should get moved inside the for loop - right now, you're always returning 1 as the day the lion got too hungry...which isn't correct. Yes, the assert is passing, but not for the right reasons. :)